resources/monitoring/charts/grafana/values.yaml

```diff
@@ -117,9 +114,6 @@ extraLabels: {}
 # priorityClassName:
 
 downloadDashboardsImage:
-  repository: curlimages/curl
```

Comment:
we are not using that image/functionality, please just comment out and do not specify in the listing

Reply:
We can leave them as they are, but that means we need to add to them to the list of exceptions.
They will not be sent to security scanning pipeline and will not trigger any error on PR checks (these checks will be enabled on kyma repo after this PR is merged). 
The problem is that security tooling has no idea about flags that you use in helm charts, so it will be hidden even if you enable it. It might be noticed on our scans that run on development clusters, but there is no guarantee for that (depends on use case).

Reply:
The image was restored